code/game/machinery/OpTable.dm

```diff
@@ -43,60 +40,63 @@
 		return TRUE
 
 /obj/machinery/optable/CanPass(atom/movable/mover, turf/target, height=0)
-	if(height==0) return 1
-
+	if(height==0)
+		return TRUE
 	if(istype(mover) && mover.checkpass(PASSTABLE))
-		return 1
+		return TRUE
 	else
-		return 0
+		return FALSE
 
 
-/obj/machinery/optable/MouseDrop_T(atom/movable/O as mob|obj, mob/user as mob)
-	if(usr.stat || (!ishuman(user) && !isrobot(user)) || user.restrained() || !check_table(user) || user.IsWeakened() || user.stunned)
```

Comment:
i think `user.IsWeakened()` and `user.stunned` got lost? I agree they logically should be part of `.incapacitated` but they don't seem to be?..
Though I'm not sure if this matters enough.

Reply:
Hmm. @SabreML said they were... Let's find out.